test/models/offer_test.rb

```diff
@@ -57,6 +47,10 @@ class OfferTest < ActiveSupport::TestCase
     assert_equal Offer.new(pay_rate_time_unit: 2).pay_rate_time_unit_key, "year"
   end
 
+  test "#rate returns properly formatted string of pay_rate_value and pay_rate_time_unit" do
+    assert_equal Offer.new(pay_rate_value: 50, pay_rate_time_unit: 2).rate, "50.0/year"
```

Comment:
One of the ideas behind enums is so that we don't reference values by ID.

```suggestion
    assert_equal Offer.new(pay_rate_value: 50, pay_rate_time_unit: :year).rate, "50.0/year"
```